content/docs/setup/kubernetes/_index.md

```diff
@@ -46,15 +46,15 @@ We recommend you install Istio for production using the
 If you run Kubernetes on a supported platform, you can follow the instructions
 specific to your Kubernetes platform:
 
-- [Alibaba Cloud Kubernetes Container Service](/docs/setup/kubernetes/install/alicloud/)
-- [Google Kubernetes Engine](/docs/setup/kubernetes/install/gke/)
-- [IBM Cloud](/docs/setup/kubernetes/install/ibm/)
+- [Alibaba Cloud Kubernetes Container Service](/docs/setup/kubernetes/install/platform/alicloud/)
+- [Google Kubernetes Engine](/docs/setup/kubernetes/install/platform/gke/)
+- [IBM Cloud](/docs/setup/kubernetes/install/platform/ibm/)
 
-If you want to install and use Istio through a Container Network Interface
-(CNI), visit our [CNI guide](/docs/setup/kubernetes/install/cni/).
+If you want your installation to use Istio's Container Network Interface
```

Comment:
Should we add a note why someone should look into CNI?  e.g. requiring Istio users to have elevated Kubernetes netadmin permissions is a problem for you, then check out CNI.

Reply:
The link, just below, explains why.